code/workspaces/web-app/src/selectors/routerSelectors.js

```diff
@@ -1,5 +1,5 @@
-const selectUrlHash = ({ router: { location: { hash } } }) => hash;
+const selectSearchHash = ({ router: { location: { search } } }) => search;
```

Comment:
Sorry to be pedantic... the search is the part of the url starting with the ?, the hash is the part of the url starting with the #, so selectSearchHash sounds like it's the part starting with ?...#, which isn't the case - see https://reactrouter.com/web/api/location.  I think this should be selectUrlSearch.

Reply:
Totally correct, my mistake